server/server.js

```diff
@@ -4,41 +4,35 @@ const passport = require('passport');
 const bcrypt = require('bcryptjs');
 const LocalStrategy = require('passport-local');
 const bodyParser = require('body-parser');
-const methodOverride = require('method-override');
 const kanban = require('./routes/kanban');
-const knex = require('./database/knex');
 const redis = require('connect-redis')(session);
 
 const User = require('./database/models/User');
 const app = express();
-const PORT = process.env.PORT || 8080;
-const ENV = process.env.NODE_ENV || 'development';
-const SESSION_SECRET = process.env.SESSION_SECRET || 'keyboard cat';
+const PORT = process.env.SERVER_PORT;
+const ENV = process.env.NODE_ENV;
+const SESSION_SECRET = process.env.SESSION_SECRET;
+const REDIS = process.env.REDIS_HOST + ':' + process.env.REDIS_HOST_PORT;
```

Comment:
Minor but I would change t he name to `REDIS_URI` to indicate explicitly that the variable is a full resource identifier